testng-core/src/test/java/test/support/SafeRandoms.java

```diff
@@ -0,0 +1,23 @@
+package test.support;
+
+import java.security.SecureRandom;
+import java.util.Random;
+
+public final class SafeRandoms {
+
+  private static final Random random = new SecureRandom();
+
+  private SafeRandoms() {}
+
+  public static int nextInt(int bound) {
+    return nextInt(random, bound);
+  }
+
+  public static int nextInt(Random random, int bound) {
```

Comment:
It is unclear what "bound" means here. Consider adding javadocs in such cases.

Reply:
It was supposed to be representing the upper bound value. Added javadocs and pushed.